test/openAPI/features/client_create.feature

```diff
@@ -1,16 +1,130 @@
+    And The POST /client-mgmt/oidc-client endpoint response should have status 200
+    And The POST /client-mgmt/oidc-client endpoint response should have content-type: application/json header
+    And The POST /client-mgmt/oidc-client endpoint response should match json schema with error
+    And The POST /client-mgmt/oidc-client endpoint response should match with error code "invalid_client_name"
+
+  @unit @negative
+  Scenario: Not able to add the new client to the Open ID Connect (OIDC) because of invalid relyingPartyId
+    Given The user wants to add the new client to the Open ID Connect (OIDC)
+    When User sends POST request with given '' as invalid relyingPartyId
+    Then User receives a response from the POST /client-mgmt/oidc-client endpoint
+    And The POST /client-mgmt/oidc-client endpoint response should be returned in a timely manner 15000ms
+    And The POST /client-mgmt/oidc-client endpoint response should have status 200
+    And The POST /client-mgmt/oidc-client endpoint response should have content-type: application/json header
+    And The POST /client-mgmt/oidc-client endpoint response should match json schema with error
+    And The POST /client-mgmt/oidc-client endpoint response should match with error code "invalid_rp_id"
+
+  @unit @negative
+  Scenario: Not able to add the new client to the Open ID Connect (OIDC) because of invalid logoUri
+    Given The user wants to add the new client to the Open ID Connect (OIDC)
+    When User sends POST request with given '' as invalid logoUri
+    Then User receives a response from the POST /client-mgmt/oidc-client endpoint
+    And The POST /client-mgmt/oidc-client endpoint response should be returned in a timely manner 15000ms
+    And The POST /client-mgmt/oidc-client endpoint response should have status 200
+    And The POST /client-mgmt/oidc-client endpoint response should have content-type: application/json header
+    And The POST /client-mgmt/oidc-client endpoint response should match json schema with error
+    And The POST /client-mgmt/oidc-client endpoint response should match with error code "invalid_uri"
+
+  @unit @negative
+  Scenario: Not able to add the new client to the Open ID Connect (OIDC) because of invalid clientId
+    Given The user wants to add the new client to the Open ID Connect (OIDC)
+    When User sends POST request with given '' as invalid clientId
+    Then User receives a response from the POST /client-mgmt/oidc-client endpoint
+    And The POST /client-mgmt/oidc-client endpoint response should be returned in a timely manner 15000ms
+    And The POST /client-mgmt/oidc-client endpoint response should have status 200
+    And The POST /client-mgmt/oidc-client endpoint response should have content-type: application/json header
+    And The POST /client-mgmt/oidc-client endpoint response should match json schema with error
+    And The POST /client-mgmt/oidc-client endpoint response should match with error code "invalid_client_id"
```

Comment:
Can we also add a scenario with empty string for redirectUris and expect invalid_redirect_uri ?

Reply:
In the API spec redirectUris is not required and can be just string so it means if I add `""` it should work. If redirectUris should have minimum 1 letter then this information has to be added do the API spec. For now there is no such information so software candidate will not know it.

Reply:
ok, in the next iteration of the spec, we will add this

Reply:
I have created the below PR for making the redirectUris manditory and waiting for review from @jdub13 on this
https://github.com/GovStackWorkingGroup/bb-identity/pull/36